erts/emulator/beam/bif_instrs.tab

```diff
@@ -37,7 +37,9 @@ CALL_GUARD_BIF(BF, TmpReg, Dst) {
     PROCESS_MAIN_CHK_LOCKS(c_p);
     ASSERT(!ERTS_PROC_IS_EXITING(c_p));
     ERTS_CHK_MBUF_SZ(c_p);
+    DEBUG_SWAPOUT;
     result = (*$BF)(c_p, $TmpReg, I);
+    DEBUG_SWAPIN;
```

Comment:
Again, maybe check that stop and htop have not moved?

Reply:
Yes. Done.